css-flexbox/justify-content.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Flexbox - justify-content</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="pokemon.css">
+  <style>
+    #card-deck {
+      display: flex;
+      flex-basis: 25;
+      padding: 0.5rem;
+    }
+
+    .card-wrapper {
+      justify-content: center;
+      padding: 0.5rem;
+    }
+
+  </style>
```

Comment:
We need to fix this styles because on bigger screen size, it's not centered horizontally